kartothek/io_components/utils.py

```diff
@@ -258,6 +257,13 @@ def _make_list(_args):
             return old_value
         else:
             return _make_list(old_value)
+    elif arg_name == "dispatch_by":
+        if old_value is None:
+            return []
+        elif isinstance(old_value, list):
+            return old_value
+        else:
+            return _make_list(old_value)
```

Comment:
Isn't this the same logic as the `_make_list`? What's the reason for this special case?

Reply:
My mistake. This should have been
```python
        if old_value is None:
            return old_value
```